### PR TITLE
replacement md5 using crypt function in README

### PR DESCRIPTION
**Pull request by hkarasek**
MD5 hash is ugly, crypt too, but less. I would like to use bcrypt, but the example must be simple and compatible with PHP 5.3.1.

### CODE REVIEW

**Comment by legendik** [comment]
How about `Nette\Security\Passwords`?

**Comment by hkarasek** [comment]
I would, but it is an example of a custom implementation.

**Comment by dg** [comment]
There should be Nette\Security\Password.

**Comment by hkarasek** [comment]
Is it okay?

File: readme.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ class MyAuthenticator extends Nette\Object implements NS\IAuthenticator
 			throw new NS\AuthenticationException('User not found.');
 		}
 
-		if ($row->password !== md5($password)) {
+		if (NS\Passwords::verify($password, $row->password)) {
```
Comment (dg) [inline readme.md]: If (!NS\Password

**Comment by dg** [comment]
Can you squash it to single commit?

**Comment by dg** [comment]
merged d42efcfa73dae0509f4b32e77ecbde298e141a72

**Pull request closed without merging**